java/meterpreter/meterpreter/src/main/java/com/metasploit/meterpreter/core/core_enumextcmd.java

```diff
@@ -0,0 +1,26 @@
+package com.metasploit.meterpreter.core;
+
+import com.metasploit.meterpreter.CommandManager;
+import com.metasploit.meterpreter.Meterpreter;
+import com.metasploit.meterpreter.TLVPacket;
+import com.metasploit.meterpreter.TLVType;
+import com.metasploit.meterpreter.command.Command;
+
+public class core_enumextcmd implements Command {
+
+    public int execute(Meterpreter meterpreter, TLVPacket request, TLVPacket response) throws Exception {
+        String command = request.getStringValue(TLVType.TLV_TYPE_STRING);
+        CommandManager commandManager = meterpreter.getCommandManager();
+        String[] commands = commandManager.getCommands();
+        for (String loadedCommand : commands) {
+            if (command.equals(loadedCommand.split("_")[0])) {
+                response.addOverflow(TLVType.TLV_TYPE_STRING, loadedCommand);
+            }
+        }
+        if ("android".equals(command)) {
```

Comment:
I tried adding a android_ prefix to the android commands, but this seemed cleaner, maybe not.